package.json

```diff
@@ -6,30 +6,41 @@
   "repository": "alex-e-leon/print-tree",
   "author": "alex leon <alex@leon.st>",
   "license": "MIT",
+  "files": [
+    "lib"
+  ],
   "scripts": {
     "build": "babel index.js -d lib/",
-    "prepublish": "yarn run build",
-    "lint": "./node_modules/.bin/eslint index.js",
-    "test:unit": "mocha --compilers js:babel-core/register test",
-    "test": "yarn run flow && yarn run lint && yarn run test:unit"
+    "prepare": "yarn build",
+    "lint": "eslint index.js test",
+    "test:watch": "ava --watch",
+    "test": "yarn flow && yarn lint && nyc ava",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov"
```

Comment:
When I try run coverage it tells me I need a repo token - did you forget to checkin coverage.lcov?

Reply:
CI will provide the token for the `coverage` command

Reply:
So - I guess we need to setup CI then?